bonfire/bonfire.py

```diff
@@ -398,6 +405,16 @@ def _validate_opposing_opts(ctx, param, value):
         default=conf.EPHEMERAL_ENV_NAME,
         show_default=True,
     ),
+    click.option(
+        "--prefer",
+        help=(
+            "Prefer deployment targets that have '<parameter name>=<value>' when duplicate"
```

Comment:
This is worded funny. I'm not sure I'd understand it if I saw it IRL. Maybe something like

```
"When there are multiple deployment targets prefer the one that has '<parameter name>=<value>' set. Can be specified multiple times."
```

I know its wordier but maybe?

Reply:
I like it, will change